Bottom: 87120d91e94d1b3ab49021d18c989e66bcc5876d
Top:    f3fee5b83c0303a36e67ed9e9d201c1103578e9e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:32:02 +0000

Refresh of fix-board-outline-generation

---

diff --git a/src/polygon.c b/src/polygon.c
index da734f8..7be99db 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2463,7 +2463,7 @@ POLYAREA *board_outline_poly (bool include_holes)
      and outside the board! */
 
   /* If there is no result, or only one piece, return that */
-  if (clipepd == NULL || clipped->f == clipped)
+  if (clipped == NULL || clipped->f == clipped)
     return clipped;
 
   /* WARNING: This next check is O(n^2), where n is the number of clipped
